telegram/ext/picklepersistence.py

```diff
@@ -123,15 +124,15 @@ def __init__(
         ...
 
     def __init__(
-        self,
-        filename: Union[Path, str],
+        self: 'PicklePersistence[UD, CD, BD]',
```

Comment:
I don't think the annotation for `self` is right here. does mypy ask for it? if not, please remove it.

Reply:
done